TerminalDocs/customize-settings/actions.md

```diff
@@ -190,7 +190,7 @@ This lets you navigate through your search matches.
 { "command": { "action": "findMatch", "direction": "prev" } }
 ```
 
-#### Actions
+#### Arguments
```

Comment:
We could also call them "Parameters". Thoughts?

Reply:
Tomato tomato. Yeah I'll update this haha